lib/apiservers/engine/backends/container.go

```diff
@@ -58,6 +58,19 @@ import (
 
 const (
 	bridgeIfaceName = "bridge"
+
+	// MemoryMinMiB - the minimum allowable container memory size
+	MemoryMinMiB = 1024
+	// MemoryMaxMiB - the maximum allowable container memory size
+	MemoryMaxMiB = 16384
+	// MemoryDefaultMiB - the default container VM memory size
+	MemoryDefaultMiB = 4096
```

Comment:
I would consider giving a user a chance to define these defaults via vic-machine during VCH deployment.